tests/it/bitmap/mutable.rs

```diff
@@ -368,7 +368,11 @@ fn extend_bitmap() {
     assert_eq!(b.len(), 5 + 4);
 }
 
+// TODO! undo miri ignore once issue is fixed in miri
+// this test was a memory hog and lead to OOM in CI
+// given enough memory it was able to pass succesfully on a local
 #[test]
```

Comment:
MIRI used a lot of memory on this one and caused OOM. 

I think we can try to turn it on later once the issue is fixed in miri.